src/locales/en.js

```diff
@@ -385,6 +385,7 @@ export default {
         back_to_relations: 'Back to relations',
         list_of_relations_feature: 'List of relations of feature',
         error_missing_father_field: "Field is missing",
+        download_with_relations: "Download with relations",
```

Comment:
Probably the same string should be added to the rest of i18n files so translators can spot new string and submit translation?